external/source/exploits/bypassuac_injection/dll/src/Exploit.cpp

```diff
@@ -46,7 +46,7 @@ extern "C" {
 				break;
 			}
 
-			if (pFileOp->SetOperationFlags(FOF_NOCONFIRMATION | FOF_NOERRORUI | FOF_SILENT | FOFX_SHOWELEVATIONPROMPT | FOFX_NOCOPYHOOKS | FOFX_REQUIREELEVATION) != S_OK)

```

Comment:
This looks like a debug change. Does this need to be here, or does it work with the original flags?

Reply:
The original flags make UAC bypass imposible from 15003 Windows 10 build, this is why I modified them in this way. This is explained in the first comment.